examples/update-on-async-rendering/initializing-state-after.js

```diff
@@ -0,0 +1,10 @@
+// After
+class ExampleComponent extends React.Component {
+  // highlight-range{1-6}
+  state = {
+    count: 0,
+    derivedValue: computeDerivedValue(
```

Comment:
How about adding

```js
// For the rare cases where initial state depends on props
```

?

Also how about replacing the whole `computeDerivedValue` thing with something like:

```js
// In rare cases, where initial state might depends on props
initialColor: this.props.color
```

Then it's both clear when you'd use this pattern, and it doesn't "conflict" in naming with `getDerivedState...` described later. Some readers will think `computeDerivedValue` is some kind of React API.